### PR TITLE
[enterprise-4.14] CNV#21651: RN - Align tekton tasks with instancestypes

### PR DESCRIPTION
**Pull request by danielclowers**
Version(s):
4.14

Issue:
https://issues.redhat.com/browse/CNV-26316

Link to docs preview:
https://65932--docspreview.netlify.app/openshift-enterprise/latest/virt/release_notes/virt-4-14-release-notes

QE review:
- [x] QE has approved this change.

Additional information:
I have included two different release notes in a single PR due to them being related, and will most likely need the same person(s) to review them both.

**Note**: I used the wrong Jira in the branch. Instead of 21651, it should be [CNV-26316](https://issues.redhat.com//browse/CNV-26316).

### CODE REVIEW

**Comment by danielclowers** [comment]
hi @geetikakay can you take a look at these two RNs?

File: virt/release_notes/virt-4-14-release-notes.adoc [hunk 1]
```diff
@@ -142,6 +145,9 @@ Deprecated features are included in the current release and supported. However,
 //CNV-26426 [DOCS] Release note: Deprecate TTO
 * The `tekton-tasks-operator` is deprecated and Tekton tasks and example pipelines are now deployed by the `ssp-operator`.
 
+//CNV-21651
+* The `copy-template`, `modify-template`, and `create-vm-from-template` tasks are deprecated.
```
Comment (geetikakay) [inline virt/release_notes/virt-4-14-release-notes.adoc:149]: modify-vm-template *
@ksimon1 Hi Karel, we are marking tasks as deprecated but there is no decision to deprecate template . For now i see only 2012 templates are mark as deprecated?

Comment (ksimon1) [reply virt/release_notes/virt-4-14-release-notes.adoc:149]: The templates will be sooner or lated deprecated. We agreed, that we will keep template based tasks marked as deprecated

Comment (geetikakay) [reply virt/release_notes/virt-4-14-release-notes.adoc:149]: @danielclowers then just one comment please rename modify-template to modify-vm-template

**Comment by geetikakay** [comment]
lgtm

**Review by jldohmann — APPROVED** [review]
LGTM

**Pull request merged**